templates/zerver/help/enable-or-disable-pressing-enter-to-send.md

```diff
@@ -1,21 +1,22 @@
 # Press Enter to send
 
-By default, Zulip sends your message when you press enter (or return)
-on your keyboard. If you want to turn this off, simply uncheck the
-**Press Enter to send** box below the compose field:
+By default, Zulip sends your message when you press the **Send** button in
+the **Compose box**. If you would like to send your message by pressing
+the enter key, follow these steps:
 
-![Box checked](/static/images/help/enter-box-checked.png)
-
-You will now see a **Send** button below the compose field looking
-like this.
+1. Check the **Press Enter to send** box below the compose field:
 
 ![Box unchecked](/static/images/help/enter-box-unchecked.png)
 
-With the box unchecked, you will have to click the **Send** button to
-send your message. Pressing enter will no longer send your message.
+2. Now press the enter key to send your message.
```

Comment:
Some computer don't have an Enter key, so you should rewrite this as "Enter (or return) key"